src/components/RuntimeManager.js

```diff
@@ -379,7 +379,7 @@ export default class RuntimeManager extends PureComponent {
           href: applicationLaunchLink,
           onClick: window.location.hash === applicationLaunchLink && currentStatus === 'Stopped' ? () => this.startRuntime() : undefined,
           tooltip: canCompute ? `Open ${applicationName}` : noCompute,
-          'aria-label': `Open ${applicationName}`,
+          ...(canCompute ? { 'aria-label': `Open ${applicationName}` } : {}),
```

Comment:
Can you combine these?

Reply:
I'm actually closing this out. Found the real issue was a missing role, which is why it was complaining about `aria-label`! (And I added that to @andrewmarcus's PR)